app/src/main/java/com/pr0gramm/app/ui/fragments/FeedFragment.java

```diff
@@ -10,25 +10,14 @@
 import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.SearchView;
 import android.util.Log;
-import android.view.Display;
-import android.view.LayoutInflater;
-import android.view.Menu;
-import android.view.MenuInflater;
-import android.view.MenuItem;
-import android.view.View;
```

Comment:
Danke für die Anmerkungen, ich werd probieren das in diesem Pull Request noch einzuarbeiten. Mal schauen wie das funzt ohne das ich nen neuen machen muss, kenne mich bei Github in diesem Punkt noch nicht wirklich aus.